### PR TITLE
Create two rounds of matchups

### PR DESCRIPTION
**Pull request by squush**
Hard to see here, but this is some of the testing for the Matchup creation. This code generates two rounds of matchups automatically when you click "Create Tournament". 
image

### CODE REVIEW

File: app/controllers/tournaments_controller.rb [hunk 1]
```diff
@@ -153,6 +154,41 @@ def create_event(event)
       xtables_id: xtables_id) unless Event.find_by(xtables_id: xtables_id)
   end
 
+  def generate_two_rounds_matchups(tournament)
+    # Check how many divisions there are
+    all_players = Player.where(tournament: tournament)
+    divisions = all_players.map { |player| player.division }.uniq
```
Comment (andygw) [inline app/controllers/tournaments_controller.rb:160]: Do we need to do this, or could we just have a num_divisions column in the tournament table?

Comment (andygw) [inline app/controllers/tournaments_controller.rb:160]: Do we need to do this, or can we just have a num_divisions column in the tournament table?

File: app/policies/matchup_policy.rb [hunk 2]
```diff
@@ -10,6 +10,10 @@ def resolve
   #   true
   # end
 
+  def create?
+    true
```
Comment (andygw) [inline app/policies/matchup_policy.rb:14]: Should only be if user is owner

File: app/controllers/tournaments_controller.rb (hunk 1 above)
Comment (squush) [reply app/controllers/tournaments_controller.rb:160]: We could do that as well, but that doesnt seem too much simpler. Maybe would be good to have eventually if I need to keep referencing this variable in other contexts, but for now it feels low priority

File: app/policies/matchup_policy.rb (hunk 2 above)
Comment (squush) [reply app/policies/matchup_policy.rb:14]: true

**Pull request merged**